service/static/js-src/muk.search.js

```diff
@@ -4044,10 +4068,20 @@ $.extend(edges, {
                     t.count = 0
                 }
             }
+            queryErr = false;
+            this.updating = false;
+            this.draw()
+        };
+
+        this.doUpdateQueryFail = function(params) {
+            for (var i in this.terms) {
+                this.terms[i].count = 0;
+            }
+            queryErr = true;
```

Comment:
Is it possible to pass this around in the params object, instead of being a global?